studen.rb

```diff
@@ -1,12 +1,19 @@
 require './person'
 
 class Student < Person
-  def initialize(age, classroom, name = 'unknown', parent_permission: true)
+  attr_reader :classroom
+
+  def initialize(age, _classroom, name = 'unknown', parent_permission: true)
     super(age, name, parent_permission: parent_permission)
-    @classroom = classroom
+    @classroom = classrniloom
```

Comment:
- [ ] It would be nice if you could correct the typo here, as this may affect your project in future milestones by causing an error.